backend/src/controllers/EventController.ts

```diff
@@ -268,6 +272,38 @@ export class EventController {
 
     return this.rsvpMapper.entityToResponse(newRSVP);
   }
+
+  @Delete('/:eventID/unrsvp/affiliate')
```

Comment:
if we are treating rsvp as an entity, then probably the following is a better endpoint name because we are DELETE-ing the rsvp
```suggestion
  @Delete('/:eventID/rsvp/affiliate')
```